### DISCUSSION TITLE
弱弱问一句，mega和lite这两个包要怎么使用？

### DISCUSSION THREAD

**Discussion by kulkatt** [hooke007/mpv_PlayKit#425] [mpv 通用]
mega解包了再加进去一个mpv.exe发现没菜单
是需要自己去编辑配置文件吗？

**Comment by hooke007** [comment]
先看必读说明。
菜单是mpv-lazy的组件。

lite mega这两是纯粹的通用的vs环境组件
默认文件结构是按照mpv-lazy设计的

https://github.com/hooke007/MPV_lazy/wiki/3_K7sfunc

**Reply by kulkatt** [reply]
谢谢，说明有点太复杂了，可能是我过于懒，还是希望mega也有直接解压就能用的包

**Reply by hooke007** [reply]
你要用什么？

**Reply by kulkatt** [reply]
其实我也不知道，看这个包有1G那么大，应该有很多好东西吧，比如滤镜，着色器什么的😂

**Reply by hooke007** [reply]
体积大写了原因：

>是纯粹的通用的vs环境组件

**Reply by hooke007** [reply]
搞清楚自己需求先，不然我也不知道你要做什么

**Reply by kulkatt** [reply]
好的O(∩_∩)O谢谢，其实只是想试试看看mega和标准版有添加什么特别的功能
